### PR TITLE
Add arm extension to the robot controller

### PR DESCRIPTION
**Pull request by porglezomp**

### CODE REVIEW

**Comment by a2aaron** [comment]
Can't merge this until we test this on the robot, however it looks good.

File: FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java [hunk 1]
```diff
@@ -247,7 +254,13 @@ public void setRightSpeed(double speed) {
 	
 	@Override
 	public void setArmSpeed(double speed) {
-		// armMotor.set(speed * armSpeed);
+		armMotor.set(speed * armSpeed);
```
Comment (a2aaron) [inline FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java:257]: Does the slave not need to be set?

Comment (porglezomp) [inline FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java:257]: There's only a single motor

**Comment by a2aaron** [comment]
Ok, approving this

**Pull request merged**